source/reference/editor-mode.txt

```diff
@@ -246,10 +246,20 @@ Using Visual Studio as an External Editor
 Visual Studio requires a special parameter to work as an external
 editor. Use ``--wait`` with Visual Studio.
 
+Set an environment variable:
+
 .. code-block:: javascript
 
    export EDITOR="/usr/local/bin/code --wait"
 
+You can also set the editor with :ref:`config.set()
+<mongosh-shell-settings>`. If Visual Studio is in your path, open
```

Comment:
nit: Does it make sense to capitalize `path` to `PATH`? It’s how the environment variable is commonly spelled and `your path` might sound a bit unclear for users who don’t know what it refers to.

Reply:
Suggestion to update this:

"If Visual Studio is in the system ``PATH`` "

Similar context is used in : 
https://docs.mongodb.com/manual/core/security-automatic-client-side-encryption/#enabling-automatic-client-side-field-level-encryption